msteams-platform/m365-apps/prerequisites.md

```diff
@@ -91,24 +91,34 @@ To install Microsoft 365 Beta Channel applications in your test environment:
 
 ### Mobile
 
-You can preview Teams personal tabs running in Microsoft 365 app for Android by joining the beta program.
+You can preview Teams personal tabs running in Microsoft 365 and Outlook app for Android by joining the beta program.
 
 To install the latest Microsoft 365 app beta, build to your physical Android device or Android emulator:
 
 1. Ensure you use a Google Play [supported Android device](https://support.google.com/googleplay/answer/1727131).
 1. Launch the **Play Store** on your Android device.
-1. Search for "Office" and select **Office (Microsoft 365)**.
+1. Search for your app of choice. For Microsoft 365 app (formerly Office), search office and select **Microsoft 365 app**. For Outlook, search outlook and select **Microsoft Outlook**.
 1. Select the **Install** button.
 
     :::image type="content" source="images/office-android-install.png" alt-text="The screenshot is an example that shows the install button for the Office (Microsoft 365) app in Google Play Store.":::
 
-1. Select **Join** under **Join the beta** section after the installation completes.
+1. For Microsoft 365 app, select **Join** under **Join the beta** section after the installation completes.
 
     :::image type="content" source="images/office-android-join-beta.png" alt-text="The screenshot is an example that shows the Join the beta screen.":::
 
 1. Launch the Microsoft 365 app and sign in with your test tenant credentials.
 1. Open your profile **(Me) > Settings** and scroll to the bottom of the menu.
-1. Ensure you use Microsoft 365 app version 16.0.15726.20000 or higher for Android.
+1. Ensure you use Microsoft 365 app version 16.0.15726.20000 or higher for Android and/or Outlook app version 4.2247.1 (52247812) or higher for Android.
+
+## Switch to the Developer Preview version of Teams
+
+Ensure that you switch to the [Public Developer Preview](../resources/dev-preview/developer-preview-intro.md) from your Microsoft Teams client.
+
+1. Sign in to Teams with your sandbox tenant credentials.
+1. From the ellipsis (**...**) menu next to your user profile, select **About** > **Developer preview**. A dialog appears, select **Switch to developer preview**.
```

Comment:
**Global comment**

Please check the right casing of Developer Preview. Please make it consistent across all.

Reply:
incorporated